### DISCUSSION TITLE
QR Code scan

### DISCUSSION THREAD

**Discussion by mnfmnc** [opengisch/QField#3024] [Q&A]
Is it possible to scan a QR code through the app, to fill a pre-built table?
Thank you!

**Comment by lucienicolier** [comment]
Hi @mnfmnc, 
It is not possible to use the QR widget yet, but the development is planned for the end of the summer,

Best regards,

**Comment by miili** [comment]
This feature has been implemented in PR https://github.com/opengisch/QField/pull/3142
